codec/src/test/java/io/netty/handler/codec/xml/XmlFrameDecoderTest.java

```diff
@@ -78,7 +75,8 @@ public void testDecodeWithInvalidInput() {
     public void testDecodeWithInvalidContentBeforeXml() {
         XmlFrameDecoder decoder = new XmlFrameDecoder(1048576);
         EmbeddedChannel ch = new EmbeddedChannel(decoder);
-        ch.writeInbound(Unpooled.copiedBuffer("invalid XML<foo/>", CharsetUtil.UTF_8));
+        ch.writeInbound(Unpooled.copiedBuffer("invalid XML<foo/>",
```

Comment:
Could you please keep the original formatting so that the changeset contains only the relevant changes?